src/proxy/http/profiles/router.rs

```diff
@@ -133,24 +133,23 @@ where
     }
 }
 
-impl<G, Inner, RouteLayer, RouteBody, InnerBody, Target, InnerSvc, RouteMake, RouteSvc>
-    svc::Service<Target> for MakeSvc<G, Inner, RouteLayer, RouteBody, InnerBody>
+impl<G, Inner, RouteLayer, RouteBody, InnerBody, Target, RouteSvc> svc::Service<Target>
```

Comment:
Nice cleanups in this file, but I don't see how it's related to the rest of the PR

Reply:
i needed to make some changes below and was fighting type signatures to make it all happy... i can probably back some of this out...